developments_build/sql/_create.sql

```diff
@@ -0,0 +1,88 @@
+DROP TABLE IF EXISTS lookup_occ;
+CREATE TABLE lookup_occ(
+    dob_occ text,
+    occ text
+); 
+\COPY lookup_occ FROM 'data/lookup_occ.csv' DELIMITER ',' CSV HEADER;
+
+
+DROP TABLE IF EXISTS lookup_ownership;
+CREATE TABLE lookup_ownership (
+    cityowned text,
+    ownertype text,
+    nonprofit text,
+    ownership text
+);
+\COPY lookup_ownership FROM 'data/lookup_ownership.csv' DELIMITER ',' CSV HEADER;
+
+
+DROP TABLE IF EXISTS housing_input_research;
```

Comment:
We'll need to update this if we merge #387

Reply:
we can update after merging to #387